src/operations/indexes.ts

```diff
@@ -72,7 +72,7 @@ export type IndexSpecification = OneOrMore<
 >;
 
 /** @public */
-export interface IndexInformationOptions extends ListIndexesOptions {
+export interface IndexInformationOptions<TFull extends boolean> extends ListIndexesOptions {
```

Comment:
We could consider this a breaking change, since existing users' code that makes use of `IndexInformationOptions` would break because they wouldn't be providing a generic argument.  We could fix this by providing a default, but I think overloads are preferable (see my comment on `Collection.indexInformation()`).